src/works/work-getters.ts

```diff
@@ -191,6 +191,11 @@ export const getWorkLocked = ($workPage: WorkPage) => {
   return !!$workPage("#signin > .heading").text();
 };
 
+export const getWorkAdult = ($workPage: WorkPage): boolean => {
+  const adultCategories: WorkRatings[] = [WorkRatings.EXPLICIT, WorkRatings.MATURE, WorkRatings.NOT_RATED];
```

Comment:
Also, I just found out about this: https://mkosir.github.io/typescript-style-guide/#type-safe-constants-with-satisfies

Apparently the best way of declaring this constant would be:
```ts
const ADULT_CATEGORIES = [
  WorkRatings.EXPLICIT, 
  WorkRatings.MATURE, 
  WorkRatings.NOT_RATED
] as const satisfies ReadonlyArray<WorkRatings>;
```
this would give the most correct typing (feel free to ask if you want to understand more).

I'd be ok with keeping it `const ADULT_CATEGORIES: WorkRatings[]`, but since you're changing it let's try the new one?